packages/bloomf/bloomf.0.2.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer:   "Clément Pascutto"
+authors:      ["Clément Pascutto"]
+license:      "MIT"
+homepage:     "https://github.com/mirage/bloomf"
+bug-reports:  "https://github.com/mirage/bloomf/issues"
+dev-repo:     "git+https://github.com/mirage/bloomf.git"
+doc:          "https://mirage.github.io/bloomf/"
+
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+  name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+  "@doc" {with-doc}
+  ]
+]
+
+depends: [
+  "ocaml"   {>= "4.03.0"}
+  "dune"    {>= "1.7.0"}
```

Comment:
```suggestion
  "dune"    {>= "2.0"}
```
Should be "2.0" according to the dune-project file: https://github.com/mirage/bloomf/blob/master/dune-project#L1

Reply:
Thanks!